### PR TITLE
Xenoarch cargo supply packs

### PR DESCRIPTION
**Pull request by JMWTurner**
Added 3 new cargo supply packs

under supplies
- floodlight flood

under science
- suspension field generator suspension
- excavation equipment crate

### CODE REVIEW

**Comment by ghost** [comment]
Emergency floodlights are good, for when power is failing and botany wants to keep its plants alive

**Comment by Killette2** [comment]
@Skullyton that is completely unrelated, but floodlights are pretty damned bad for that. Floodlights have a battery, and if we're talking about the box map then the library has some really bright desk lamps next door. Desk lamps and flashlights are a much better alternative that doesn't take a whole tile.

**Comment by JMWTurner** [comment]
Just wait until we buff their light range.

**Comment by Killette2** [comment]
Sounds good, know if putting in a 15k cell hikes up their charge like when you replace the default cell to a stun baton?

**Comment by JMWTurner** [comment]
of course it does. the wonders of modular code.

**Comment by jknpj** [comment]
Good. Now miners and traders can get their hand-pickaxes without breaking the windo- GOD DAMN WHO DESPRESSURIZED THE RESEARCH OUTPOST AGAIN

**Comment by SkowronX** [comment]
This reminds me. Can floodlights be opened to change their cells?

**Comment by JMWTurner** [comment]
floodlight

yeah

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -331,6 +331,14 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	containername = "EFTPOS crate"
 	group = "Supplies"
 
+/datum/supply_packs/floodlight
+	name = "Emergency Floodlight crate"
+	contains = list(/obj/machinery/floodlight)
+	cost = 5
```
Comment (ComicIronic) [inline code/datums/supplypacks.dm]: Should be much higher.

File: code/datums/supplypacks.dm [hunk 2]
```diff
@@ -1378,6 +1386,34 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	access = access_robotics
 	group = "Science"
 
+/datum/supply_packs/suspension_gen
+	name = "Suspension Field Generator crate"
+	contains = list(/obj/machinery/suspension_gen)
+	cost = 20
```
Comment (ComicIronic) [inline code/datums/supplypacks.dm]: Can afford to be much higher.

File: code/datums/supplypacks.dm [hunk 3]
```diff
@@ -1378,6 +1386,34 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
...
+		/obj/item/weapon/storage/box/excavation,
+		/obj/item/device/flashlight/lantern,
+		/obj/item/device/depth_scanner,
+		/obj/item/device/core_sampler,
+		/obj/item/clothing/glasses/scanner/meson,
+		/obj/item/weapon/pickaxe,
+		/obj/item/device/measuring_tape,
+		/obj/item/weapon/pickaxe/hand,
+		)
+	cost = 10
```
Comment (ComicIronic) [inline code/datums/supplypacks.dm]: Seriously?

**Comment by JMWTurner** [comment]
revision 1
- balanced prices

File: code/datums/supplypacks.dm [hunk 4]
```diff
@@ -1378,6 +1386,34 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
...
+/datum/supply_packs/suspension_gen
+	name = "Suspension Field Generator crate"
+	contains = list(/obj/machinery/suspension_gen)
+	cost = 50
+	containertype = /obj/structure/largecrate
+	containername = "suspension field generator crate"
+	access = access_research
+	group = "Science"
+
+/datum/supply_packs/suspension_gen
```
Comment (ComicIronic) [inline code/datums/supplypacks.dm]: This type is duplicated - did you test these crates?

**Comment by ComicIronic** [comment]
Still not really happy about these prices - emergency lighting for 20 credits is pretty cheap, especially considering you can get 3 boxes of lightbulbs for 10.

File: code/datums/supplypacks.dm (hunk 4 above)
Comment (JMWTurner) [inline code/datums/supplypacks.dm]: I'm never smoking weed while coding ever again.

**Comment by JMWTurner** [comment]
I made those prices so they'd fit the other packs. I think the low prices of most packs are explained by the fact there were no salaries back then. If prices are all you're concerned about now I can make another PR to review each pack's price one by one and balance them all.

Fueltanks are 8$ for crying out loud.

**Pull request merged**